microsite/siteConfig.js

```diff
@@ -93,8 +93,10 @@ const siteConfig = {
   cleanUrl: true,
 
   // Open Graph and Twitter card images.
-  ogImage: 'img/logo-gradient-on-dark.svg',
-  twitterImage: 'img/logo-gradient-on-dark.svg',
+  ogImage:
+    'logo_assets/png/Backstage Identity_Assets_Artwork_RGB_04 Icon_Teal.png',
```

Comment:
`%20` for the spaces perhaps? then again - i would really prefer if these filenames didn't have spaces in them to begin with 🙅

Reply:
@freben Yeah, I preferred the same. I think spaces should be removed from all the images filename. All the images from `logo_assets` have spaces in there filename.